graylog2-web-interface/src/components/configurations/PermissionsConfig.tsx

```diff
@@ -0,0 +1,138 @@
+  &&.deflist {
+    dd {
+      padding-left: ${theme.spacings.md};
+      margin-left: 200px;
+    }
+  }
+`);
+
+const LabelSpan = styled.span(({ theme }: { theme: DefaultTheme }) => css`
+  margin-left: ${theme.spacings.sm};
+  font-weight: bold;
+`);
+
+const PermissionsConfig = ({ config, updateConfig }: Props) => {
+  const [showModal, setShowModal] = useState<boolean>(false);
+
+  const _saveConfig = (values) => {
+    updateConfig(values).then(() => {
+      setShowModal(false);
+    });
+  };
+
+  const _resetConfig = () => {
+    setShowModal(false);
+  };
+
+  return (
+    <div>
+      <h2>Permissions Config</h2>
+      <p>These settings can be used to control which entity sharing options are available</p>
+
+      {!config ? <Spinner /> : (
+        <>
+          <StyledDefList>
+            <dt>Share with everyone:</dt>
+            <dd>{config.allow_sharing_with_everyone ? 'Enabled' : 'Disabled'}</dd>
+            <dt>Share with users:</dt>
+            <dd>{config.allow_sharing_with_users ? 'Enabled' : 'Disabled'}</dd>
+          </StyledDefList>
+
+          <p>
+            <Button type="button"
+                    bsSize="xs"
+                    bsStyle="info"
+                    onClick={() => {
+                      setShowModal(true);
+                    }}>Configure
+            </Button>
+          </p>
+
+          <Modal show={showModal} onHide={_resetConfig} aria-modal="true" aria-labelledby="dialog_label">
+            <Formik onSubmit={_saveConfig} initialValues={config}>
+
+              {({ isSubmitting }) => {
+                return (
+                  <Form>
+                    <Modal.Header closeButton>
+                      <Modal.Title id="dialog_label">Configure Permissions</Modal.Title>
+                    </Modal.Header>
+
+                    <Modal.Body>
+                      <div>
+                        <Row>
+                          <Col sm={12}>
+                            <FormikInput type="checkbox"
+                                         name="allow_sharing_with_everyone"
+                                         id="shareWithEveryone"
+                                         label={(
+                                           <LabelSpan>Share with everyone</LabelSpan>
+                                         )} />
+                            <InputDescription help={<>Share with everyone option will be available</>} />
+                          </Col>
+                          <Col sm={12}>
+                            <FormikInput type="checkbox"
+                                         name="allow_sharing_with_users"
+                                         id="shareWithUsers"
+                                         label={(
+                                           <LabelSpan>Share with users</LabelSpan>
+                                         )} />
+                            <InputDescription help={<>Share with users options will be available.</>} />
```

Comment:
Same as above, I would suggest something like "Control whether it is possible to share with single users".